.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:34
+
+# Install few packages for convenient usage.
+# Inspired by the devcontainer in osbuild.
+RUN dnf install -y \
+    fish \
+    fd-find \
+    ripgrep \
+    jq
+# We build composer using RPM, install the tooling
+RUN dnf install fedora-packager rpmdevtools -y
+# The list of requirements is specified in the spec file.
+# To install them, dnf needs the "builddep" command.
+RUN dnf install 'dnf-command(builddep)' -y
+# Copy the specfile, remove gometa because dnf fails
```

Comment:
Just install `go-srpm-macros` before running `dnf builddep`, and then you don't need this workaround. I also wonder if we should merge all `dnf install` commands into one `RUN`.

Reply:
Nice, I forgot about the `go-srpm-macros` 🤦 . I decided to split those `RUN`s to make it more structured and also to have a place for writing comments, but I'm also fine with a single giant `RUN dnf install`

Reply:
This comment is unnecessary now. :)